moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -999,55 +985,55 @@ bool RobotState::clearAttachedBody(const std::string& id)
     return false;
 }
 
-const Eigen::Isometry3d& RobotState::getFrameTransform(const std::string& id)
+const Eigen::Isometry3d& RobotState::getFrameTransform(const std::string& frame_id)
```

Comment:
These changes should clear up if you rebase your branch on master.

Reply:
Thanks, worked.